__tests__/index.test.js

```diff
@@ -42,4 +35,121 @@ describe('GET /api', () => {
             expect(endpoints).toEqual(jsonEndpoints);
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('GET: 200 returns the article id that is input into the endpoint', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article;
+            expect(article.article_id).toEqual(1);
+        })
+    })
+    test('GET: 200 returns the article id that is input into the endpoint', () => {
+        return request(app)
```

Comment:
This test seems to be identical to the above except for checking a different number?